pkg/datapath/loader/base.go

```diff
@@ -418,6 +392,27 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 		log.Warning("Cannot check matching of C and Go common struct alignments due to old LLVM/clang version")
 	}
 
+	// Compile and load network programs, currently used for encryption.
+	if option.Config.EnableIPSec {
+		interfaces := option.Config.EncryptInterface
+
+		// For the ENI ipam mode on EKS, this will be the interface that
+		// the router (cilium_host) IP is associated to.
+		if option.Config.IPAM == ipamOption.IPAMENI {
+			if len(option.Config.IPv4PodSubnets) == 0 {
+				if info := node.GetRouterInfo(); info != nil {
+					for _, c := range info.GetIPv4CIDRs() {
+						option.Config.IPv4PodSubnets = append(option.Config.IPv4PodSubnets, &c)
+					}
+				}
+				log.Info("Encryption IPv4PodSubnets in-use")
```

Comment:
This should be printed once on startup at most right? Not once every time we reinitialize the datapath? (Or at Debug level?)

Reply:
Lets just remove it this is a bit pointless.